frontend/packages/dev-console/integration-tests/features/guided-tour/quick-starts/quick_start_card_on_cluster_overview_ODC-4922.feature

```diff
@@ -0,0 +1,53 @@
+Feature: Add Quick Starts card on cluster overview
+	As a user, I should be able to access and discover Quick Starts from the Cluster Overview in the Administrator view
+
+Background:
+    Given user is at administrator perspective
+    And user has created CR for sample-application Quick Start
+    And user has created CR for install-associate-pipeline Quick Start
+    And user has created CR for explore-pipeline Quick Start
+    And user has created CR for add-healthchecks Quick Start
+
+@regression
+Scenario: Quick Starts card on Cluster Overview 
+   Given user is in administrator perspective
+   When user goes to Cluster Overview page
+   Then user can see Quick Starts card 
+   And user can see three Quick Starts link present on it
+   And user can see the "View all Quick Starts" 
+   And user can see the kebab menu on top right of the card
+
+@regression
+Scenario: Quick Starts links with status as in progress 
+   Given user is at Cluster Overview page
+   When user clicks on first Quick Starts link on the Quick Starts card 
+   And user clicks on the Start tour
+   And user clicks on close button
+   And user clicks on leave on leave the tour modal box
+   Then user can see "In Progress" status below the first Quick Starts link
+ 
+@regression
+Scenario: Quick Starts card when one Quick Starts has completed
+   Given user is at Cluster Overview page
+   When user completes first Quick Starts from the card
+   Then user can see completed Quick Starts link is removed from the card
+ 
+@regression
+Scenario: Quick Starts card when all Quick Starts has completed
+   Given user is at Cluster Overview page
+   When user completes all four Quick Starts present
+   Then user can see Quick Starts card is removed from the Cluster Overview page
```

Comment:
Actually when all the Quick Starts are completed the card will be removed from both the perspectives

Reply:
I'll add that scenario in the developer perspective file, this is just for the cluster overview page